app/scrapers/football_scraper.py

```diff
@@ -0,0 +1,315 @@
+            ).text
+        ):
+            self.seasons_links.append(
+                self.season_link_unformatted.format(season_number=current_season_number)
+            )
+            logger.info(
+                f"Added new season: {current_season_number - 62}/{str(current_season_number - 61)[-2:]}"
+            )
+
+            current_season_number += 1
+
+        logger.info(f"Finished to build season links, total seasons: {len(self.seasons_links)}")
+        return self.seasons_links
+
+    def fetch_matches_from_site(
+        self, url_to_fetch: str | None = None, to_update_last_match: bool | None = False
+    ) -> list[CalendarEvent]:
+        """
+        Gets matches from the official site, parse them and return as list of events
+
+        :param url_to_fetch: Optional. The URL to web scrap from. Default is upcoming matches URL
+        :param to_update_last_match: Optional. A flag to indicate if to get only the last played match or all events
+
+        :return: a list of events representing the matches
+        """
+        if not url_to_fetch:
+            url_to_fetch = self.upcoming_matches_url
+        # Connect to the URL
+        response = httpx.get(url_to_fetch)
+        response.raise_for_status()
+
+        # Parse HTML and save to BeautifulSoup object
+        soup = BeautifulSoup(response.text, "html.parser")
+
+        events: list[CalendarEvent] = []
+
+        if not to_update_last_match:
+            unparsed_matches = soup.find_all("div", {"class": "fixtures-holder"})
+            logger.info(f'List of {len(unparsed_matches)} parsed matches:"')
+
+            for match_data in unparsed_matches:
+                # Ignoring matches without final schedule or U19 league
+                if (
+                    isinstance(match_data, bs4.element.Tag)
+                    and match_data.find(text="מועד לא סופי") is None
+                    and match_data.find(text="לנוער") is None
+                ):
+                    event = self.handle_match(match_data)
+                    if "לנוער" in event.description:
+                        logger.debug(f"Ignoring U19 match: {event}")
+                        continue
+                    events.append(event)
+                else:
+                    logger.debug("\n=============================================================")
+                    logger.debug(f"Ignoring match without final date or a U19 match: {match_data}")
+                    logger.debug("\n=============================================================")
+        else:
+            # Getting last match result
+            match_data = soup.find("div", {"class": "fixtures-holder"})
+            if isinstance(match_data, bs4.element.Tag):
+                event = self.handle_match(match_data)
+                events.append(event)
+            else:
+                logger.warning("Unable to find last match data or data is not a valid Tag")
```

Comment:
That happened? I don't see this in the old code

Reply:
לא ברור, ראיתי שחלק מהמשחקים ביומן הישן לא עודכנו כראוי, אז הוספתי את הלוג הזה למקרה ונרצה לתחקר בעתיד

Reply:
נשמע לי סבבה, אבל אולי נזרוק ]ה שגיאה? כי זה לא אמור לקרות